Control/Applicative/Combinators/NonEmpty.hs

```diff
@@ -0,0 +1,49 @@
+-- |
+-- This module provides non-empty list variants of some of the functions in
+-- "Control.Applicative.Combinators" and re-exports `some1` from
+-- "Data.List.NonEmpty".
```

Comment:
Add `@since 0.2.0` matadata to the header, so it's clear in which version it was added.